### PR TITLE
[Fix-6139][UI] fix some UI bugs

### PR DESCRIPTION
**Pull request by wangyizhi1**
#6139

Fix the following problems:

1. For the reference node problem, there should be no need to change after reference, and the search box should be at the top. Deselection (temporarily hidden) should be supported after selection

2. The task node supports viewing versions (not implemented temporarily)

3. Workflow version deletion, 404 problem

4. The projectid in the dependent node needs to be replaced with projectcode, and the definitionlist field needs to be removed

========

1、引用节点问题，引用之后应无需更改，而且搜索框应该在最上面，选择之后应支持取消选择（暂时隐藏）
2、任务节点支持查看版本（暂时不实现）
3、工作流版本删除，404问题
4、依赖节点中 projectId 需要换成 projectCode，去除definitionList字段

_Originally posted by @lenboo in https://github.com/apache/dolphinscheduler/issues/6139#issuecomment-919655795_

5. Missing icon in tis switch

6. The parameter is missing 'environmentcode'. The default value is - 1
===========

5  TIS/SWITCH任务缺少图标
6  参数缺少 environmentCode，默认值-1

_Originally posted by @lenboo in https://github.com/apache/dolphinscheduler/issues/6139#issuecomment-919656618_

7 Click in the workflow instance to enter the sub workflow, and an interface call error occurs
http://*:12345/dolphinscheduler/#/projects/724559144984576/instance/list/undefined?subProcessCodes=726631487152128

http://*:12345/dolphinscheduler/#/projects/724559144984576/instance/list/undefined?subProcessCodes=726631487152128

===============

7 工作流实例中点击进入子工作流，出现接口调用错误

_Originally posted by @lenboo in https://github.com/apache/dolphinscheduler/issues/6139#issuecomment-919691872_

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/_source/dependItemList.vue [hunk 1]
```diff
@@ -215,7 +215,7 @@
 
       _cpOldParams (value, definitionCode, definitionList, depTasksList, item) {
         return {
-          projectId: value,
+          projectCode: value,
           definitionList: definitionList,
```
Comment (brave-lee) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/_source/dependItemList.vue:219]: The definitionlist field needs to be removed in the dependent node, pleae check

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/dependent.vue [hunk 2]
```diff
@@ -119,7 +119,7 @@
           dependTaskList: _.map(this.dependTaskList, v => {
             return {
               relation: v.relation,
-              dependItemList: _.map(v.dependItemList, v1 => _.omit(v1, ['depTasksList', 'state', 'dateValueList']))
+              dependItemList: _.map(v.dependItemList, v1 => _.omit(v1, ['depTasksList', 'state', 'dateValueList', 'definitionList']))
             }
```
Comment (wangyizhi1) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/dependent.vue:123]: omit definitionList

**Review by wangyizhi1 — COMMENTED** [review]
@JinyLeeChina 

The `definitionList` will be filtered out here.

**Comment by break60** [comment]
dolphinscheduler-ui/src/js/module/i18n/locale/en_US.js
image
130 lines and 132 exchange the next order
Line 131 is not quoted, please delete

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/dependent.vue (hunk 2 above)
Comment (brave-lee) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/dependent.vue:123]: good job

**Comment by lenboo** [comment]
> dolphinscheduler-ui/src/js/module/i18n/locale/en_US.js
> image
> 130 lines and 132 exchange the next order
> Line 131 is not quoted, please delete

i notice there is a '+' at the end of line 130.

**Review by lenboo — APPROVED** [review]
+1

**Pull request merged**